src/pudl/workspace/setup.py

```diff
@@ -110,7 +112,11 @@ def derive_paths(pudl_in, pudl_out):
     pudl_settings["pudl_out"] = str(pudl_out)
     # One directory per output format:
     for fmt in ["sqlite", "parquet"]:
-        pudl_settings[f"{fmt}_dir"] = str(pudl_out / fmt)
+        pudl_settings[f"{fmt}_dir"] = str(pudl_out)
```

Comment:
Should we add a deprecation warning here about the sqlite and parquet directories?